capture.py

```diff
@@ -416,6 +421,17 @@ def _isolated_nodes(nodes, panel):
             cmds.isolateSelect(panel, addDagObject=obj)
     yield
 
+@contextlib.contextmanager
+def _maintained_time():
+    """Context manager for preserving (resetting) the time after the context"""
+    from maya import cmds
```

Comment:
Hmm, I wonder why it's being imported in the function.

Do you think we should import it at the top of the file instead?